chalice/app.pyi

```diff
@@ -161,6 +161,18 @@ class DecoratorAPI(object):
                        batch_size: int=1,
                        name: Optional[str]=None) -> Callable[..., Any]: ...
 
+    def on_kinesis_message(self,
```

Comment:
How about on `on_kinesis_record` and `on_dynamodb_record` instead? In the documentation that seems to be the terminology:
* Kinesis: https://docs.aws.amazon.com/lambda/latest/dg/with-kinesis.html
* DynamoDB: https://docs.aws.amazon.com/lambda/latest/dg/with-ddb.html

Reply:
Yeah I'm not opposed to that.  The event payload uses `Records` for all the event types (including SNS and SQS) but at least for kinesis, the API call is `PutRecord` so I think `on_kinesis_record` makes sense.  For DynamoDB, unless we're going to break completely from the existing naming schemes and do something like `on_dynamodb_change` or `on_dynamodb_activity`, etc.  I think it makes sense to stick with either `message` or `record`.

Reply:
Yeah it definitely makes sense for Kinesis. I only suggested it also for DynamoDB because that is what the docs seem to favor instead of `message` (which is used in the docs for SQS/SNS).